modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -6,36 +6,17 @@
 [id="nodes-pods-user-namespaces-configuring_{context}"]
 = Configuring Linux user namespace support
 
+You can configure your pods to run in Linux user namespaces by setting the `hostUsers` parameter to `false` in the workload spec, as shown in the following procedure. 
 
-.Prerequisites
+Running workloads in user namespaces allows you to specify `RunAsAny` for Security Context Constraint (SCC) fields, such as `fsGroup`, `runAsGroup`, `runAsUser`, and `supplementalGroups`. Setting these fields to `RunAsAny` can have specific benefits described throughout the {product-title} documentation. Within a user namespace, `RunAsAny` is generally considered to be a safe practice. For extra security, you can create a new SCC specifically for workloads in user namespaces by setting the `userNamespaceLevel: RequirePodLevel` field in the SCC, which requires that the workloads run in user namespaces. To require a specific SCC, set the `openshift.io/required-scc` annotation in any workload resource that can set a pod manifest template, such as a deployment or daemon set.
 
-* You enabled the required Technology Preview features for your cluster by editing the `FeatureGate` CR named `cluster`:
-+
-[source,terminal]
-----
-$ oc edit featuregate cluster
-----
-+
-.Example `FeatureGate` CR
-[source,yaml]
-----
-apiVersion: config.openshift.io/v1
-kind: FeatureGate
-metadata:
-  name: cluster
-spec:
-  featureSet: TechPreviewNoUpgrade <1>
-----
-<1> Enables the required `UserNamespacesSupport` and `ProcMountType` features.
-+
-[WARNING]
-====
-Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. This feature set allows you to enable these Technology Preview features on test clusters, where you can fully test them. Do not enable this feature set on production clusters.
-====
-+
-After you save the changes, new machine configs are created, the machine config pools are updated, and scheduling on each node is disabled while the change is being applied.
+Also, configuring workloads as `procMount: unmasked` is generally considered as safe within a user namespace. Setting `procMount` to `unmasked` has benefits that are beyond the scope of this documentation. 
+
+To ensure that the namespace functionality exists on all nodes that you want to run in a user namespace, you can configure the minimum version of kubelet that is required for the nodes in your cluster. If the kubelet version in your cluster is lower than this version, new nodes are not scheduled and existing nodes are marked as degraded. For existing nodes with a lower version, the kubelet can only read the node object by using `oc get` or `oc update` commands, and the actions that the node can perform, by using a `SelfSubjectAccessReview`. The node is not allowed to gain access to any other API objects.     
```

Comment:
> For existing nodes with a lower version, the kubelet can only read the node object by using `oc get` or `oc update` commands

the kubelet doesn't use `oc` , you can just say the kubelet can only read and update its own node object

Reply:
@haircommander 
> actually minimumKubeletVersion cannot be set if the current nodes in the cluster are too old.

How old to "too old"?

Reply:
older than the defined MinimumKubeletVersion

Reply:
@haircommander 

> For existing nodes with a lower version, the kubelet can only read its node object.

Is this statement true only if I set a minimum kubelet version? Are there any negative ramifications that the user who runs lower kubelet version(s) should be aware of?

Reply:
> Is this statement true only if I set a minimum kubelet version

yeah

> Are there any negative ramifications that the user who runs lower kubelet version(s) should be aware of?

There are two cases:
- if the lower kubleet version exists in the cluster at the time min kubelet version is attempted to be set, the validation rejects the min kubelet update
- if the node is being added after the min kubelet version is established, that node will not be able to connect to the cluster meaningfully. It would take manual intervention to fix